workspaces/stack-overflow/plugins/stack-overflow/README.md

```diff
@@ -1,12 +1,18 @@
 # Stack Overflow
 
-A plugin that provides stack overflow specific functionality that can be used in different ways (e.g. for homepage and search) to compose your Backstage App.
+A [Backstage](https://backstage.io/) plugin that integrates Stack Overflow content into your Backstage app. Use it to surface relevant Stack Overflow questions and answers in search results or display them on your homepage.
 
-## Getting started
+> Note: For Stack Overflow specific search results to be returned, it needs to be indexed. Use the [Stack Overflow Search Backend Module](https://github.com/backstage/backstage/blob/master/plugins/search-backend-module-stack-overflow-collator/README.md) to index Stack Overflow Questions to search.
```

Comment:
Sorry for extra work but should we move this like we did for the Explore Plugin's collator recently?